x-pack/platform/plugins/private/observability_ai_assistant_management/public/routes/components/settings_tab/change_kb_model.tsx

```diff
@@ -199,6 +199,7 @@ export function ChangeKbModel({
     if (error) {
       return (
         <EuiCallOut
+          announceOnMount={false}
```

Comment:
It appears conditionally after an async load, so it’s dynamic. I think we need to add `announceOnMount={true}` here.